isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp

```diff
@@ -1009,6 +1042,43 @@ namespace Isis {
     int numUsed;
     int imageIndex = 0;
 
+    char imgMeasuresH1[] = "        Measures        ";
+    char imgMeasuresH1a[]= "************************";
+    char imgMeasuresH2[] = "|Accepted    |    Total|";
+    char imgMeasuresH3[] = "             RMS(pixels)             ";
+    char imgMeasuresH3a[] ="*************************************";
+    char imgMeasuresH4[] = "|Samples    |    Lines    |    Total|";
+
+
+
```

Comment:
Why so many spaces here?

Reply:
This entire section of code has been refactored.  Check it again.